src/content/docs/apm/agents/ruby-agent/getting-started/migration-9x-guide.mdx

```diff
@@ -236,6 +236,20 @@ See the [milestone for 9.0](https://github.com/newrelic/newrelic-ruby-agent/mile
   The configuration option `instrumentation.thread.tracing` is now enabled by default. This configuration allows the agent to properly monitor code occurring inside threads. In Ruby agent 9.0, instrumented code within threads will be recorded and associated with the current transaction when the thread was created.
 This may be a breaking change if you're currently using custom thread instrumentation. New transactions inside of threads will no longer be started if one already exists.
 
+  Asynchronous segments may not be finished before the current transaction finishes. This can cause segments to have inaccurate timing information. In version 9.0.0 of the agent, a `NoMethodError` may be raised in these scenarios while a transaction finishes: 
```

Comment:
```suggestion
  Asynchronous segments may not be finished before the current transaction finishes. This can cause segments to have inaccurate timing information. In Ruby agent 9.0.0, a `NoMethodError` may be raised in these scenarios while a transaction finishes: 
```